src/Parser/Hunter/Marksmanship/Modules/Talents/CarefulAim.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox from 'Interface/Others/StatisticBox';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+import { formatNumber } from 'common/format';
+import STATISTIC_ORDER from 'Interface/Others/STATISTIC_ORDER';
+
+/**
+ * Aimed Shot has a 50% chance to deal 100% bonus damage to targets who are above 80% health or below 20% health.
+ *
+ * Example log: https://www.warcraftlogs.com/reports/v6nrtTxNKGDmYJXy#fight=16&type=auras&source=6
+ */
+const HIGHER_HP_THRESHOLD = 0.8;
+const LOWER_HP_THRESHOLD = 0.2;
+const AIMED_AP_MOD = 2.07;
+const MASTERY_DMG_MODIFIER_PER_PERCENT = 2.24;
```

Comment:
Is the mastery modifier different on AS than other shots? Otherwise it would probably be better to update the mastery coef instead.

Reply:
Our mastery % is range on abilities, our damage increase is 2.24 times as large.